### PR TITLE
DOCS-3658 Adds Use Session Replay In Your Solutions/Support Workflows Guide + Troubleshooting Doc Page

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Extracts troubleshooting and FAQs from Session Replay index page into a Troubleshooting doc page, adds screenshots and content for the Use Session Replay In Your Solutions/Support Workflows guide. 

### Motivation

DOCS-3658 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 1]
```diff
@@ -70,85 +76,22 @@ if (user.isAuthenticated) {
...
 
-### How do you disable Session Replay?
+### Disable Session Replay
 
-- Remove `startSessionReplayRecording()` to stop session recordings.
-- Set `replaySampleRate` to `0` to stop collecting the Browser Premium plan for RUM & Session Replay, which includes replays, resources, and long tasks.
+To stop session recordings, remove `startSessionReplayRecording()` and set `replaySampleRate` to `0`. This stops collecting data for the [Browser Premium plan][7] for RUM & Session Replay, which includes replays, resources, and long tasks.
 
-In order to apply these configurations, upgrade the [Browser RUM SDK][2] to a version >= 3.6.
+In order to apply these configurations, upgrade the [Browser RUM SDK][5] to a version >= 3.6.
```
Comment (alai97) [inline content/en/real_user_monitoring/session_replay/_index.md:85]:
```suggestion
In order to apply these configurations, upgrade the [RUM Browser SDK][5] to a version >= 3.6.
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 2]
```diff
@@ -70,85 +76,22 @@ if (user.isAuthenticated) {
...
-
-Session Replay follows the same 30 days retention policy as normal RUM sessions.
+To stop the Session Replay recording, call `stopSessionReplayRecording()`.
 
-### How do you disable Session Replay?
+### Disable Session Replay
 
-- Remove `startSessionReplayRecording()` to stop session recordings.
-- Set `replaySampleRate` to `0` to stop collecting the Browser Premium plan for RUM & Session Replay, which includes replays, resources, and long tasks.
+To stop session recordings, remove `startSessionReplayRecording()` and set `replaySampleRate` to `0`. This stops collecting data for the [Browser Premium plan][7] for RUM & Session Replay, which includes replays, resources, and long tasks.
```
Comment (alai97) [inline content/en/real_user_monitoring/session_replay/_index.md:83]:
```suggestion
To stop session recordings, remove `startSessionReplayRecording()` and set `replaySampleRate` to `0`. This stops collecting data for the RUM & Session Replay's [Browser Premium plan][7], which includes replays, resources, and long tasks.
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 3]
```diff
@@ -52,15 +60,13 @@ datadogRum.init({
...
+### CDN
 
-The Session Replay recorder supports all browsers supported by the RUM Browser SDK with the exception of IE11. For more information, see the [browser support table][3].
+Replace the RUM Browser SDK URL `https://www.datadoghq-browser-agent.com/datadog-rum.js` with `https://www.datadoghq-browser-agent.com/datadog-rum-v4.js`. When `DD_RUM.init()` is called, the Session Replay recording does not start until `DD_RUM.startSessionReplayRecording()` is also called.
 
-### Configuration
+## Configuration
 
-The usual [RUM initialization parameters][4] are all supported.
+The usual [RUM initialization parameters][6] are all supported.
```
Comment (alai97) [inline content/en/real_user_monitoring/session_replay/_index.md:69]:
```suggestion
You can use the RUM Browser SDK's [initialization parameters][6].
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 4]
```diff
@@ -70,85 +76,22 @@ if (user.isAuthenticated) {
...
-
-Session Replay follows the same 30 days retention policy as normal RUM sessions.
+To stop the Session Replay recording, call `stopSessionReplayRecording()`.
 
-### How do you disable Session Replay?
+### Disable Session Replay
 
-- Remove `startSessionReplayRecording()` to stop session recordings.
-- Set `replaySampleRate` to `0` to stop collecting the Browser Premium plan for RUM & Session Replay, which includes replays, resources, and long tasks.
+To stop session recordings, remove `startSessionReplayRecording()` and set `replaySampleRate` to `0`. This stops collecting data for the RUM & Session Replay's [Browser Premium plan][7], which includes replays, resources, and long tasks.
```
Comment (alai97) [inline content/en/real_user_monitoring/session_replay/_index.md:83]:
```suggestion
To stop session recordings, remove `startSessionReplayRecording()` and set `replaySampleRate` to `0`. This stops collecting data for RUM & Session Replay's [Browser Premium plan][7], which includes replays, resources, and long tasks.
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 5]
```diff
@@ -1,38 +1,46 @@
...
+The Session Replay recorder supports all browsers supported by the RUM Browser SDK with the exception of IE11. For more information, see the [browser support table][3]. 
 
-#### npm
+In order to reduce Session Replay's network impact and ensure the Session Replay recorder has minimal overhead on your application's performance, Datadog compresses the data prior to sending it, and reduces the load on a browser’s UI thread by delegating most of the CPU-intensive work (such as compression) to a background service worker. The expected network bandwidth impact is less than 100Kb/min. 
 
-Replace the `@datadog/browser-rum package` with a version >3.6.0 of [`@datadog/browser-rum`][2]. To start the recording, call `datadogRum.startSessionReplayRecording()`.
+## Setup
+
+Session Replay is available through a dedicated build of the RUM Browser SDK. To start collecting data for Session Replay, set up [Datadog RUM Browser Monitoring][4] by creating a RUM application, generating a client token generation, and initializing the RUM Browser SDK.
```
Comment (acorretti) [inline content/en/real_user_monitoring/session_replay/_index.md:35]:
```suggestion
Session Replay is available in the RUM Browser SDK. To start collecting data for Session Replay, set up [Datadog RUM Browser Monitoring][4] by creating a RUM application, generating a client token generation, and initializing the RUM Browser SDK.
```
Just checked with @DataDog/rum-browser and this sentence isn't accurate anymore. There is no dedicated build, SR is included in the standard bundle/package now.

**Comment by kayayarai** [comment]
created https://datadoghq.atlassian.net/browse/DOCS-3697 to track editorial review

File: content/en/real_user_monitoring/session_replay/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+### CSS rules not properly applied/mouse hover not replayed
+
+Unlike fonts and images, the recorder tries to bundle the various CSS rules applied as part of the recording data, leveraging the [CSSStyleSheet][5] interface. If this is not possible, it falls back to recording the links to the CSS files.
+
+For proper mouse hover support, the CSS rules must be accessible through the CSSStyleSheet interface.
+
+If the stylesheets are hosted on a different domain than the web page, access to the CSS rules is subject to cross-origin security checks by the browser, and the browser must be instructed to load the stylesheet leveraging CORS using the [crossorigin][6] attribute.
+
+For example, if your application is on the `example.com` domain and depends on a CSS file on `assets.example.com` via a link element, the `crossorigin` attribute should be set to `anonymous`, unless credentials are required:
```
Comment (alai97) [inline content/en/real_user_monitoring/session_replay/troubleshooting.md:44]:
```suggestion
For example, if your application is on the `example.com` domain and depends on a CSS file on `assets.example.com` through a link element, the `crossorigin` attribute should be set to `anonymous`, unless credentials are required:
```

File: content/en/real_user_monitoring/guide/session-replay-for-solutions.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+You can enable your technical solutions and support teams to better troubleshoot customer issues using [Session Replay][1]. With RUM & Session Replay, you can locate specific user sessions, observe user journeys, and access developer tools to see events, logs, errors, and attributes. 
+
+This guide describes a workflow that organizations can replicate and use as an asset for solutions teams to integrate into their workflows.
+
+{{< img src="real_user_monitoring/guide/session-replay/session-replay-recording.png" alt="Session Replay recording of a user session in the Shopist application" style="width:100%;">}}
+
+## Assess user issues
+
+Let’s assume that a customer encounters an issue using Datadog. Your Technical Solutions team may use a support solution such as Zendesk or ServiceNow that creates a ticket when this customer reports that they cannot update or save a Synthetics multistep API test.
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/session-replay-for-solutions.md:27]:
```suggestion
Assume that a customer encounters an issue using Datadog. Your Technical Solutions team may use a support solution, such as Zendesk or ServiceNow, that creates a ticket when this customer reports that they cannot update or save a Synthetics multistep API test. 
```

File: content/en/real_user_monitoring/guide/session-replay-for-solutions.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+This guide describes a workflow that organizations can replicate and use as an asset for solutions teams to integrate into their workflows.
+
+{{< img src="real_user_monitoring/guide/session-replay/session-replay-recording.png" alt="Session Replay recording of a user session in the Shopist application" style="width:100%;">}}
+
+## Assess user issues
+
+Let’s assume that a customer encounters an issue using Datadog. Your Technical Solutions team may use a support solution such as Zendesk or ServiceNow that creates a ticket when this customer reports that they cannot update or save a Synthetics multistep API test. 
+
+The team may request more information from the customer (such as the specific test ID and a screen recording with the [Browser Dev Tools][2] open) in order to obtain some additional context into the customer's test not updating or saving. If no console errors were recorded, the team would not have any hints to start investigating the multistep API test's issue.
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/session-replay-for-solutions.md:29]:
```suggestion
The team may request more information from the customer (such as the specific test ID and a screen recording with the [Browser Dev Tools][2] open) that may provide additional context for the customer's test not updating or saving. If no console errors were recorded, the team would not have any hints to start investigating the multistep API test's issue. 
```

File: content/en/real_user_monitoring/guide/session-replay-for-solutions.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+{{< img src="real_user_monitoring/guide/session-replay/apm-traces-in-session-replay.png" alt="An APM stack trace associated with a RUM view action" style="width:100%;">}}
+
+With the APM integration, you can connect requests from your web application with corresponding backend traces to access APM trace data from a RUM event and uncover any backend errors in the **Errors** tab. 
+
+For more information, see [Connect RUM and Traces][3].
+
+## Watch user sessions in Session Replay
+
+The Technical Solutions team may have internal tools that connect a support platform, like Zendesk, with Datadog products, such as RUM & Session Replay. For example, if you click on a contextual link in Zendesk, it redirects you to the [RUM Explorer][4] and autofills the user ID in the search query. You can then filter for individual user sessions in the event list.
```
Comment (cswatt) [inline content/en/real_user_monitoring/guide/session-replay-for-solutions.md:49]:
```suggestion
The Technical Solutions team may have internal tools that connect a support platform, like Zendesk, with Datadog products, such as RUM & Session Replay. For example, a contextual link in Zendesk can redirects you to the [RUM Explorer][4] and autofill the user ID in the search query. You can then filter for individual user sessions in the event list.
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 10]
```diff
@@ -1,38 +1,46 @@
...
 ## Overview
 
-Session Replay expands your user experience monitoring by allowing you to capture and visually replay the web browsing experience of your users.
+Session Replay expands your user experience monitoring by allowing you to capture and visually replay the web browsing experience of your users. Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and provides insights into your web application’s usage patterns and design pitfalls.
 
-Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and provides insights into your web application’s usage patterns and design pitfalls.
+The RUM Browser SDK is [open source][1] and leverages the open source [rrweb][2] project.
 
-## Collect Session Replay data
+## Session Replay Recorder
```
Comment (cswatt) [inline content/en/real_user_monitoring/session_replay/_index.md:23]:
```suggestion
## Session Replay recorder
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 11]
```diff
@@ -1,38 +1,46 @@
...
 
-### Initial setup
+The Session Replay recorder, part of the RUM Browser SDK, takes a snapshot of the browser's DOM and CSS by tailing and recording events happening on a web page (such as DOM modification, mouse move, clicks, and input events) along with their timestamps.
```
Comment (cswatt) [inline content/en/real_user_monitoring/session_replay/_index.md:25]:
```suggestion
The Session Replay recorder is part of the RUM Browser SDK. The recorder takes a snapshot of the browser's DOM and CSS by tailing and recording events happening on a web page (such as DOM modification, mouse move, clicks, and input events) along with these events' timestamps. 
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 12]
```diff
@@ -1,38 +1,46 @@
...
 
-To use Session Replay, set up [Datadog RUM Browser Monitoring][1]. Set up the following sections: application creation, client token generation, and RUM SDK setup.
+Datadog then rebuilds the web page and re-applies the recorded events at the appropriate time in the replay view. Session Replay follows the same 30 day retention policy as normal RUM sessions. 
 
-Session Replay is available through a dedicated build of the RUM Browser SDK. To enable Session Replay, change the npm package name or CDN URL, depending on your chosen installation method:
+The Session Replay recorder supports all browsers supported by the RUM Browser SDK with the exception of IE11. For more information, see the [browser support table][3]. 
 
-#### npm
+In order to reduce Session Replay's network impact and ensure the Session Replay recorder has minimal overhead on your application's performance, Datadog compresses the data prior to sending it, and reduces the load on a browser’s UI thread by delegating most of the CPU-intensive work (such as compression) to a background service worker. The expected network bandwidth impact is less than 100Kb/min.
```
Comment (cswatt) [inline content/en/real_user_monitoring/session_replay/_index.md:31]:
```suggestion
To reduce Session Replay's network impact and ensure the Session Replay recorder has minimal overhead on your application's performance, Datadog compresses the data prior to sending it. Datadog also reduces the load on a browser’s UI thread by delegating most of the CPU-intensive work (such as compression) to a background service worker. The expected network bandwidth impact is less than 100kB/min. 
```

File: content/en/real_user_monitoring/session_replay/troubleshooting.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+## Overview
+
+If you experience unexpected behavior with Datadog Session Replay, use this page to help resolve issues. If you continue to have trouble, contact [Datadog Support][1] for further assistance. Regularly update to the latest version of the [RUM Browser SDK][2], as each release contains improvements and fixes.
+
+## Session Replay Recorder
+
+### Some HTML elements are not visible at replay
+
+Session Replay does not support the following HTML elements at the moment: `iframe`, `video`, `audio`, `canvas`, as well as Web Components.
```
Comment (cswatt) [inline content/en/real_user_monitoring/session_replay/troubleshooting.md:22]:
```suggestion
Session Replay does not support the following HTML elements: `iframe`, `video`, `audio`, `canvas`. Session Replay does not support Web Components.
```

File: content/en/real_user_monitoring/session_replay/troubleshooting.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+
+### Fonts or images do not render properly
+
+A Session Replay is not a video, but an actual iframe rebuilt based on snapshots of the DOM. The replay thus depends on the various assets of the page: fonts & images.
```
Comment (cswatt) [inline content/en/real_user_monitoring/session_replay/troubleshooting.md:26]:
```suggestion
A Session Replay is not a video, but an actual iframe rebuilt based on snapshots of the DOM. The replay thus depends on the various assets of the page: fonts and images.
```

File: content/en/real_user_monitoring/session_replay/troubleshooting.md [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+
+Several reasons might explain why assets are not available at the time of the replay.
```
Comment (cswatt) [inline content/en/real_user_monitoring/session_replay/troubleshooting.md:28]:
```suggestion
Assets may not be available at the time of replay for the following reasons:
```

File: content/en/real_user_monitoring/session_replay/troubleshooting.md [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+
+1. The resource does not exist anymore. For example, it was part of a previous deployment.
+2. The resource is inaccessible. For example, authentication might be required, or the resource might only be accessible from an internal network.
+3. The resource is blocked by the browser due to CORS (typically web-fonts).
+   - The replay rendered on the `session-replay-datadoghq.com` sandbox domain and the asset requests are subject to cross-origin security checks by your browser. If the given asset is not authorized for the domain, your browser blocks the request.
+   - Allow `session-replay-datadoghq.com` through the [`Access-Control-Allow-Origin`][3] header for any font or image assets your website depends on to ensure these resources are accessible for the replay. For more information, see [Cross Origin Resource Sharing][4].
```
Comment (cswatt) [inline content/en/real_user_monitoring/session_replay/troubleshooting.md:34]:
```suggestion
- The resource no longer exists. For example, it was part of a previous deployment.
- The resource is inaccessible. For example, authentication might be required, or the resource might only be accessible from an internal network.
- The resource is blocked by the browser due to CORS (typically web-fonts).
   - The replay rendered on the `session-replay-datadoghq.com` sandbox domain and the asset requests are subject to cross-origin security checks by your browser. If the given asset is not authorized for the domain, your browser blocks the request.
   - Allow `session-replay-datadoghq.com` through the [`Access-Control-Allow-Origin`][3] header for any font or image assets your website depends on to ensure these resources are accessible for the replay. For more information, see [Cross Origin Resource Sharing][4].
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Review by cswatt — APPROVED** [review]

File: content/en/real_user_monitoring/_index.md [hunk 17]
```diff
@@ -6,36 +6,36 @@ disable_sidebar: true
...
-    text: "Get better RUM data with our custom React components"
-  - link: "/real_user_monitoring/browser/data_collected/"
-    tag: "Documentation"
-    text: "RUM Browser Data Collected"
-  - link: "https://www.datadoghq.com/blog/hybrid-app-monitoring/"
-    tag: "Blog"
-    text: "Monitor your hybrid mobile applications with Datadog"
+- link: "https://app.datadoghq.com/release-notes?category=Real%20User%20Monitoring"
+  tag: "Release Notes"
+  text: "Check out the latest Datadog RUM releases! (App login required)."
```
Comment (alai97) [inline content/en/real_user_monitoring/_index.md:11]:
```suggestion
  text: "Check out the latest Datadog RUM releases! (App login required)"
```

**Pull request merged**